ci/teams/rhtap-servicerelease/release-build.yaml

```diff
@@ -10,4 +10,12 @@ spec:
   application: build
   displayName: ''
   origin: rhtap-build-tenant
-  releaseStrategy: push-to-external-registry-strategy
+  releaseStrategy: rhtap-service-push-strategy
+  extraData:
```

Comment:
and we have slack notification enabled for build only?

Reply:
to start yes. gitops does not have a webhook defined in slack secret.